src/theme/components/link.ts

```diff
@@ -1,6 +1,9 @@
 export const baseLinkStyles = {
   color: "ui.link.primary",
   textDecoration: "underline",
+  _dark: {
```

Comment:
Also add dark mode colors for `_hover`.

Reply:
In the Figma file, the color in the hover state is the same as the normal state, but I'm guessing it should be "dark.ui.link.secondary" so I updated to that. I also made added dark mode for the button variant based on the dark mode of the button component. Let me know if it looks right to you when I push!

Reply:
Looks great.